<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NineHoles</class>
 <widget class="QMainWindow" name="NineHoles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NineHoles</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	background: url(':/grid');
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="Hole" name="hole00">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Hole" name="hole01">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="Hole" name="hole02">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>0</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Hole" name="hole10">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="Hole" name="hole11">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="Hole" name="hole12">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>1</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="Hole" name="hole20">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="Hole" name="hole21">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="Hole" name="hole22">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	outline: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="NineHoles.qrc">
        <normaloff>:/empty</normaloff>:/empty</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>49</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="row" stdset="0">
       <number>2</number>
      </property>
      <property name="col" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJogo">
    <property name="title">
     <string>Jogo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="actionAjuda"/>
   </widget>
   <addaction name="menuJogo"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionAjuda">
   <property name="text">
    <string>Ajuda</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Hole</class>
   <extends>QPushButton</extends>
   <header location="global">hole.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="NineHoles.qrc"/>
 </resources>
 <connections/>
</ui>
